example/scalalib/native/3-multi-module/build.mill

```diff
@@ -0,0 +1,114 @@
+package build
+import mill._, scalalib._, scalanativelib._
+
+trait MyModule extends ScalaNativeModule {
+  def scalaVersion = "2.13.11"
+  def scalaNativeVersion = "0.5.5"
+
+  object test extends ScalaNativeTests {
+    def ivyDeps = Agg(ivy"com.lihaoyi::utest::0.8.4")
+    def testFramework = "utest.runner.Framework"
+  }
+
+  // Additional source folder to put C sources
+  def nativeSources = Task.Sources(millSourcePath / "resources/scala-native")
+
+  def nativeLinkingOptions = Seq("-L" + Task.dest.toString)
+  //def barSource = millSourcePath.toString + "/resources/scala-native"
+
+  // Compile C
+  def nativeCompiled = Task {
```

Comment:
Oh... Interesting. Do need need for `nativeCompiled` since the c files are in `resources/scala-native`. Noted

About the other part, you mind explaining what you meant?

Reply:
@lihaoyi do you know what he meant at the last about installing a library globally/

Reply:
@c0d33ngr It probably means using `curl` or some native library that is installed on the operating system rather than compiled locally. I think for this PR let's skip that for now, we can do that in a follow up

Reply:
Okay... I've seen an example code with `curl`